src/core/js/models/componentModel.js

```diff
@@ -1,186 +1,182 @@
+    return 'blocks';
+  }
 
-    /**
-     * Returns a string of the model type group.
-     * @returns {string}
-     */
-    getTypeGroup() {
-      return 'component';
-    }
+  get _siblings() {
+    Adapt.log.deprecated('componentModel._siblings, use componentModel.getSiblings() instead, sibling models are defined by the JSON');
+    return 'components';
+  }
 
-    defaults() {
-      return AdaptModel.resultExtend('defaults', {
-        _isA11yComponentDescriptionEnabled: true,
-        _userAnswer: null,
-        _attemptStates: null
-      });
-    }
+  /**
+   * Returns a string of the model type group.
+   * @returns {string}
+   */
+  getTypeGroup() {
+    return 'component';
+  }
 
-    trackable() {
-      return AdaptModel.resultExtend('trackable', [
-        '_userAnswer',
-        '_attemptStates'
-      ]);
-    }
+  defaults() {
+    return AdaptModel.resultExtend('defaults', {
+      _isA11yComponentDescriptionEnabled: true,
+      _userAnswer: null,
+      _attemptStates: null
+    });
+  }
 
-    trackableType() {
-      return AdaptModel.resultExtend('trackableType', [
-        Array,
-        Array
-      ]);
-    }
+  trackable() {
+    return AdaptModel.resultExtend('trackable', [
+      '_userAnswer',
+      '_attemptStates'
+    ]);
+  }
 
-    get hasManagedChildren() {
-      return false;
-    }
+  trackableType() {
+    return AdaptModel.resultExtend('trackableType', [
+      Array,
+      Array
+    ]);
+  }
 
-    init() {
-      if (Adapt.get('_isStarted')) {
-        this.onAdaptInitialize();
-        return;
-      }
-      this.listenToOnce(Adapt, 'adapt:initialize', this.onAdaptInitialize);
-    }
+  get hasManagedChildren() {
+    return false;
+  }
 
-    onAdaptInitialize() {
-      this.restoreUserAnswers();
+  init() {
+    if (Adapt.get('_isStarted')) {
+      this.onAdaptInitialize();
+      return;
     }
+    this.listenToOnce(Adapt, 'adapt:initialize', this.onAdaptInitialize);
+  }
 
-    /**
-     * Restore the user's answer from the _userAnswer property.
-     * The _userAnswer value must be in the form of arrays containing
-     * numbers, booleans or arrays only.
-     */
-    restoreUserAnswers() {}
-
-    /**
-     * Store the user's answer in the _userAnswer property.
-     * The _userAnswer value must be in the form of arrays containing
-     * numbers, booleans or arrays only.
-     */
-    storeUserAnswer() {}
-
-    resetUserAnswer() {
-      this.set('_userAnswer', null);
-    }
+  onAdaptInitialize() {
+    this.restoreUserAnswers();
+  }
 
-    reset(type, force) {
-      if (!this.get('_canReset') && !force) return;
-      this.resetUserAnswer();
-      super.reset(type, force);
-    }
+  /**
+   * Restore the user's answer from the _userAnswer property.
+   * The _userAnswer value must be in the form of arrays containing
+   * numbers, booleans or arrays only.
+   */
+  restoreUserAnswers() {}
```

Comment:
could put a space in between the curly bracket for `restoreUserAnswers() { }` and `storeUserAnswer() { }`

Reply:
Same as above. I think this might be a personal preference thing?